docs/_toc.scss

```diff
@@ -0,0 +1,28 @@
+// table of contents
```

Comment:
migrated from `docs.less`

* renamed `.article-toc` to `.toc`
* tweaked to support potential top-of-page TOC